lang/smaily-for-wp.pot

```diff
@@ -2,10 +2,10 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: Smaily for WP\n"
-"POT-Creation-Date: 2021-02-03 12:57+0200\n"
+"POT-Creation-Date: 2021-02-18 11:10+0200\n"
 "PO-Revision-Date: 2021-01-05 20:12+0200\n"
 "Last-Translator: Smaily <info@smaily.com>\n"
-"Language-Team: Smaily <info@smaily.com>\n"
+"Language-Team: Smaily <language@example.com>\n"
```

Comment:
If you'd like the `Language-Team` value to be undetermined then change the name to something like `Language Team` also.

Reply:
Changed it back c8a47da